codersdk/templateversions.go

```diff
@@ -21,8 +21,7 @@ type TemplateVersion struct {
 	Name           string         `json:"name"`
 	Job            ProvisionerJob `json:"job"`
 	Readme         string         `json:"readme"`
-	CreatedByID    uuid.UUID      `json:"created_by_id"`
-	CreatedByName  string         `json:"created_by_name"`
+	CreatedBy      User           `json:"created_by"`
```

Comment:
I'd make this `Creator` instead

Reply:
Do you have a strong opinion about that? I like the `CreatedBy` tho.